selenium-exercise/7.py

```diff
@@ -0,0 +1,35 @@
+
+from selenium.webdriver import Chrome
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from selenium.common.exceptions import WebDriverException
+from selenium.webdriver.support.ui import Select
+
+from time import sleep
+
+driver_path = r'C:\Users\sawan\Desktop\infuse\Practice-Vineet\selenium-exercise\drivers\chromedriver-win64\chromedriver.exe'
+def select_ith_item_from_dropdown(i):
+    try:
+        chrome_service = ChromeService(executable_path=driver_path)
```

Comment:
This function is doing quite a lot of things, modularise it so its more readable.